controller.py

```diff
@@ -14,28 +18,27 @@
 class Controller:
 
     # requestQueue is shared queue among all processes
-    #@logged_method
```

Comment:
You'll see that I removed **a lot** of the `@logged_method` decorators. We can add them back in very easily where they will be useful for tracking down issues, but having them on every method adds a lot of log clutter and can make things harder to find.